events/janus_rabbitmqevh.c

```diff
@@ -377,7 +374,11 @@ int janus_rabbitmqevh_init(const char *config_path) {
 		g_free((char *)ssl_key_file);
```

Comment:
Are you sure we can free vhost, credentials and certificate stuff in the "success" case? I'm not super familiar with the library, so I don't know if it makes its own internal copy of those strings: I do know we were freeing the host before, so that should be fine. I'll make a couple of tests to see if it works, even though my RabbitMQ server is not configured with credentials or SSL support.

Reply:
I did not run, but I checked the call sequence.
The pointer falls into openssl and does not resides anywhere within rabbit:
'ssl_key_file' ->  amqp_ssl_socket_set_key() -> SSL_CTX_use_PrivateKey_file()